config/config.go

```diff
@@ -2,3 +2,46 @@
 // Basic configuration like consul credentials and address, http port to listen for requests,
 // postgres schema name, credentials, and client timeout are read from environment variables.
 package config
+
+import (
+	"strings"
+
+	"github.com/lvl484/user-manager/logger"
+	"github.com/spf13/viper"
```

Comment:
Maybe, will be better to insert empty line between "github.com/lvl484/user-manager/logger" and "github.com/spf13/viper". What do you think about it?

Reply:
Fixed

Reply:
fixed